templates/kafka/0/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+zk:
+  tty: true
+  image: rawmind/rancher-zk:0.0.1-1
```

Comment:
This version has more features. Zk cluster get reconfigured dinamically when scale up or down.